criteria.go

```diff
@@ -156,6 +156,33 @@ func (f *field) In(values ...interface{}) *Criteria {
 	}
 }
 
+func (f *field) Contains(elems ...interface{}) *Criteria {
+	return &Criteria{
+		p: func(doc *Document) bool {
+			docValue := doc.Get(f.name)
+			if docValue == nil || reflect.TypeOf(docValue).Kind() != reflect.Slice {
```

Comment:
Use type assertion instead of reflection. Documents are normalized before being inserted to db, so fields containing slices are always []interface.
```go
s, _ := docField.([]interface{})
if docField == nil || s == nil {
    return false
}
```